examples/todo/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[[bin]]
+name = "dsync"
+path = "src/dsync.rs"
+
+[[bin]]
+name = "example"
+path = "src/example.rs"
+
+[package]
+name = "todo"
+version = "0.1.0"
+edition = "2021"
+default-run = "example"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
```suggestion
```

i am pretty sure we do not need the boilerplate here, or was this explicitly added?

Reply:
I think this was the result of `cargo new` :)